File: taskapp-api/src/task/app/CalculationService.java
package task.app;

import java.io.InputStream;
import java.util.List;

import byps.Remote;
import byps.RemoteException;

/**
 * @since 1.0.0.1
 */
public interface CalculationService extends Remote {

	public int computeSimpleChecksum(List<InputStream> streams) throws RemoteException;

}

File: taskapp-ser/src-cs/task/app/CalculationService.cs
using System;
using System.Collections.Generic;
using byps;

namespace task.app
{
	
	// checkpoint byps.gen.cs.GenRemoteClass:93
	public interface CalculationService : BRemote {
	
		int ComputeSimpleChecksum(IList<System.IO.Stream> streams);
		void ComputeSimpleChecksum(IList<System.IO.Stream> streams, BAsyncResult<int> asyncResult) ;
		
		
	}
}  // end namespace
